modules/ROOT/pages/latest-overview-concept.adoc

```diff
@@ -136,6 +136,12 @@ API assets, published by Exchange, are the components that comprise applications
 
 For details about sharing assets via a private Exchange or an Exchange public portal, see xref:exchange::about-sharing-assets.adoc[Share Assets].
 
+=== API Console
+
+When creating or editing APIs in API Manager, you can access the xref:apikit::apikit-simulate[API Console] to view and simulate your API specification. This feature is available for RAML or OAS APIs with Mule v4.x, when you manage the API as an endpoint with proxy.
+
+The API Console access from API Manager is disabled by default. However, you can enable the access also modify the API Console path. Optionally, you can entirely remove the console flow from the proxy XML file if you don't need it. The console flow is already included in the API specification when you download an out-of-the-box proxy for your Mule application.
```

Comment:
```suggestion
The API Console access from API Manager is disabled by default. However, you can enable the access and also modify the API Console path. Optionally, you can entirely remove the console flow from the proxy XML file if you don't need it. The console flow is already included in the API specification when you download an out-of-the-box proxy for your Mule application.
```

Reply:
Also, this part:
```
Optionally, you can entirely remove the console flow from the proxy XML file if you don't need it
```
Is kind of redundant, considering the paragraph starts mentioning that the console is disabled by default.

Reply:
@NachoRaik Are completely removing the flow from the proxy XML file and disabling the feature the same thing?

Reply:
Yes, when disabled API Manager takes care of removing the flow from the XML. Right, @tutecano1995 ?

Reply:
Yep, as @NachoRaik mentioned, disabling the API Console from the UI and removing the flow from the XML is equivalent. 

With API Console Configuration, when you download the proxy:
- If API Console  is disabled you will not see the flow in the XML
- If the console path is modified you will see the new console path in the flow
- If the console path is the default (`/console/*`) you will see the default value in the flow

The behavior before this new feature was always having the API Console enabled with `/console/*`

Reply:
Ok, great thanks! I understood the rest but the part where disabling from the UI means that the flow is removed was not clear. I'll make the change accordingly. Thanks!

Reply:
Done! Thanks!